google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerImplTest.java

```diff
@@ -70,4 +71,49 @@ public void createAndCloseSession() {
     // The same channelHint is passed for deleteSession (contained in "options").
     Mockito.verify(rpc).deleteSession(sessionName, options.getValue());
   }
+
+  @Test
+  public void getDbclientAgainGivesSame() {
+    Map<String, String> labels = new HashMap<>();
+    labels.put("env", "dev");
+    Mockito.when(spannerOptions.getSessionLabels()).thenReturn(labels);
+    String dbName = "projects/p1/instances/i1/databases/d1";
+    DatabaseId db = DatabaseId.of(dbName);
+
+    Mockito.when(spannerOptions.getTransportOptions())
+        .thenReturn(GrpcTransportOptions.newBuilder().build());
+    Mockito.when(spannerOptions.getSessionPoolOptions())
+        .thenReturn(SessionPoolOptions.newBuilder().build());
+
+    DatabaseClient databaseClient = impl.getDatabaseClient(db);
+
+    // Get db client again
+    DatabaseClient databaseClient1 = impl.getDatabaseClient(db);
+
+    assertThat(databaseClient == databaseClient1);
```

Comment:
I don't think this actually asserts anything, I think you have to `assertThat(databaseClient1).isSameAs(databaseClient)`

Reply:
Will do, thanks.